foreman/foreman.spec

```diff
@@ -31,6 +31,14 @@ Source7: %{name}-plugins.repo
 Source8: %{name}.gpg
 BuildArch:  noarch
 
+Requires: %{name}-core = %{version}-%{release}
+Requires: %{name}-debug
+Requires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.8.5
```

Comment:
My intention is to make sure tasks are installed by default for foreman runtime, which is something else, than installing foreman for building purposes, which is what this changes try to solve and from the comments it seemed like the issue